scrapy/pqueues.py

```diff
@@ -47,9 +38,7 @@ class ScrapyPriorityQueue:
 
     startprios is a sequence of priorities to start with. If the queue was
     previously closed leaving some priority buckets non-empty, those priorities
-    should be passed in startprios.
-
-    """
+    should be passed in startprios."""
```

Comment:
:lipstick:  The separate """ on a new line was in line with https://www.python.org/dev/peps/pep-0257/